securesystemslib/signer/_key.py

```diff
@@ -56,6 +58,11 @@
 
 logger = logging.getLogger(__name__)
 
+
+class UnsupportedKeyType(Exception):  # noqa: N818
+    pass
```

Comment:
this could be marked private (_UnsupportedKeyType) since I don't think we intend to leak it outside... but that's a nit